plugins/auth/index.js

```diff
@@ -39,22 +41,28 @@ exports.register = (server, options, next) => {
         admins: joi.array().default([]),
         scm: joi.object().required()
     }), 'Invalid config for plugin-auth');
+    const scmContexts = server.app.userFactory.scm.getScmContexts();
```

Comment:
I found the reason and solution. You should use `server.root.app` here.
https://github.com/hapijs/discuss/issues/182#issuecomment-160307205